pyproject.toml

```diff
@@ -1,3 +1,134 @@
+    "Programming Language :: Python :: 3 :: Only",
+]
+dependencies = [
+    "asyncpg==0.29.*",
+    "attrs==23.2.*",
+    "boto3>=1.34,<1.36",
+    "certifi==2024.7.4",
+    "databricks-sdk==0.44.1",
+    "dataclasses-json==0.6.*",
+    "dj-database-url==2.1.0",
+    "django-cors-headers==4.3.*",
+    "django-debug-toolbar==4.3.*",
+    "django-extensions==3.2.*",
+    "django-spaghetti-and-meatballs==0.4.*",
+    "Django==4.2.*",
+    "Django==4.2.*",
+    "django_cte==1.3.*",
+    "djangorestframework==3.15.*",
+    "docutils>=0.10,<0.17",
+    "drf-api-tracking==1.8.4",
+    "drf-extensions==0.7.*",
+    "elasticsearch-dsl==7.4.*",
+    "elasticsearch==7.10.*",
+    "et-xmlfile==1.1.0",
+    "filelock==3.13.1",
+    "fiscalyear==0.4.0",
+    "Markdown==3.5.*",
+    "marshmallow==3.21.1",
+    "numpy==1.26.*",
+    "openpyxl==3.1.*",
+    "pandas==2.2.*",
+    "psutil==5.9.*",
+    "psycopg2==2.9.9",  # Pinning exact version because this package will drop support for Python versions in patches
+    "py-gfm==2.0.0",
+    "pydantic[dotenv]==1.9.*",
+    "python-json-logger==2.0.7",
+    "requests==2.31.*",
+    "retrying==1.3.4",
+    "urllib3==1.26.*",
+
+    # Opentelemetry Dependencies
+    "opentelemetry-api==1.28.2",
+    "opentelemetry-distro==0.49b2",
+    "opentelemetry-exporter-otlp==1.28.2",
+    "opentelemetry-exporter-otlp-proto-common==1.28.2",
+    "opentelemetry-exporter-otlp-proto-grpc==1.28.2",
+    "opentelemetry-exporter-otlp-proto-http==1.28.2",
+    "opentelemetry-instrumentation==0.49b2",
+    "opentelemetry-instrumentation-asgi==0.49b2",
+    "opentelemetry-instrumentation-asyncio==0.49b2",
+    "opentelemetry-instrumentation-asyncpg==0.49b2",
+    "opentelemetry-instrumentation-boto3sqs==0.49b2",
+    "opentelemetry-instrumentation-botocore==0.49b2",
+    "opentelemetry-instrumentation-dbapi==0.49b2",
+    "opentelemetry-instrumentation-django==0.49b2",
+    "opentelemetry-instrumentation-elasticsearch==0.49b2",
+    "opentelemetry-instrumentation-grpc==0.49b2",
+    "opentelemetry-instrumentation-logging==0.49b2",
+    "opentelemetry-instrumentation-psycopg2==0.49b2",
+    "opentelemetry-instrumentation-redis==0.49b2",
+    "opentelemetry-instrumentation-requests==0.49b2",
+    "opentelemetry-instrumentation-sqlite3==0.49b2",
+    "opentelemetry-instrumentation-system-metrics==0.49b2",
+    "opentelemetry-instrumentation-threading==0.49b2",
+    "opentelemetry-instrumentation-tortoiseorm==0.49b2",
+    "opentelemetry-instrumentation-urllib==0.49b2",
+    "opentelemetry-instrumentation-urllib3==0.49b2",
+    "opentelemetry-instrumentation-wsgi==0.49b2",
+    "opentelemetry-propagator-aws-xray==1.0.1",
+    "opentelemetry-proto==1.28.2",
+    "opentelemetry-sdk==1.28.2",
+    "opentelemetry-semantic-conventions==0.49b2",
+    "opentelemetry-util-http==0.49b2",
+    # The above packages are listed and pinned to avoid using the 'opentelemetry-bootstrap -a install' command
+    "xlrd3==1.1.0",
```

Comment:
If you happen to need to do any further cleanup can we move this above the opentelemetry packages? It isn't something you changed but seems out of place down here.